errors.go

```diff
@@ -9,6 +9,8 @@ import (
 var (
 	// ErrItemNotFound is returned when the item is not found in the collection.
 	ErrItemNotFound = errors.New("directus: item not found")
+	// ErrIEmpty is returned when the item is returned empty.
```

Comment:
Línea en blanco antes, y:

```suggestion
	// ErrIEmpty is returned when the item is not returned back in a create or update operation.
```